/home/rsb63/.conda/envs/gnsde/lib/python3.10/site-packages/torch_geometric/data/in_memory_dataset.py:300: UserWarning: It is not recommended to directly access the internal storage format `data` of an 'InMemoryDataset'. If you are absolutely certain what you are doing, access the internal storage via `InMemoryDataset._data` instead to suppress this warning. Alternatively, you can access stacked individual attributes of every graph via `dataset.{attr_name}`.
  warnings.warn(msg)
/home/rsb63/.conda/envs/gnsde/lib/python3.10/site-packages/torchdiffeq/_impl/misc.py:11: UserWarning: RK4: Unexpected arguments {'max_iters': 100}
  warnings.warn('{}: Unexpected arguments {}'.format(solver.__class__.__name__, unused_kwargs))
/home/rsb63/.conda/envs/gnsde/lib/python3.10/site-packages/torchdiffeq/_impl/misc.py:11: UserWarning: EarlyStopRK4: Unexpected arguments {'max_iters': 100}
  warnings.warn('{}: Unexpected arguments {}'.format(solver.__class__.__name__, unused_kwargs))
slurmstepd: error: poll(): Bad address
/var/spool/slurm/slurmd/job48551105/slurm_script: line 51: 194452 Killed                  python3 run_standard_gnsde.py --dataset Cora --epoch 100 --rtol 0.01 --sde_t 14.0 --sigma 1.0 --task_index $SLURM_ARRAY_TASK_ID
